### DISCUSSION TITLE
jtac frequency on kneeboard ?

### DISCUSSION THREAD

**Discussion by HILOK** [Khopa/dcs_liberation#1818] [Q&A] [answered]
i get no reply, when calling jatc as given on the kneeboard page 2, any ideas ? thanks

![image](https://user-images.githubusercontent.com/76898536/143860951-0d4544f9-1cc7-4ed6-90b7-eb0ca5a9bb14.png)

**Comment by Goliathus-21** [comment, marked as answer]
Hi Hilok, 

I'm not sure if you're still struggling with this, but I reported same issue as a bug. Well it seems like it isn't and it is mostly a user problem. 

I was told that JTAC operates on FM frequency, and in my case I tried to reach the JTAC on preset AM frequency. 

Check this solution, maybe it is your case as well. 

Its possible that kneeboard will be updated to show that JTAC operates in FM frequency.

**Reply by HILOK** [reply]
hi thanks for the hint, much appreciated! gonna try it. when you reached it, did you get replies from DCS's native jtac or ciribob's jtac plugin replying over SRS/STTS? thanks.

**Reply by HILOK** [reply]
yep **FM did the trick** ! worked on hornet at least. just to answer my other question above: this applies to contact DCS's **native JTAC**.

again thanks : )
